pkg/types/aws/machinepool.go

```diff
@@ -78,8 +78,7 @@ func (a *MachinePool) Set(required *MachinePool) {
 
 // EC2RootVolume defines the storage for an ec2 instance.
 type EC2RootVolume struct {
-	// IOPS defines the amount of provisioned IOPS. This is only valid
-	// for type io1.
+	// IOPS defines the amount of provisioned IOPS. (KiB/s)
```

Comment:
```suggestion
	// IOPS defines the amount of provisioned IOPS. (KiB/s). IOPS may only be set for io1, io2, & gp3 volume types.
```

Reply:
@barbacbd I'm available to work on this - what docs need revising?

Reply:
> https://docs.openshift.com/container-platform/4.10/storage/dynamic-provisioning.html This is the first instance of the issue (not sure if there are more). In the section "AWS Elastic Block Store (EBS) object definition".
> 
> In #2 GP3 is a missing value. In #3 It says that iops is only allowed for io1. It is allowed for gp3 and io2. It is required for io1 and io2 to be a value greater than 0.

@lpettyjo Brent has identified some docs inconsistencies in the storage area. Could you take a look? unsure if we should have a separate BZ created or how it should be handled. Thank you

Reply:
@barbacbd I'd like to get this comment updated, because I don't think there's anywhere in the codebase that documents this behavior.

Reply:
(And after the comment is updated the explain code will need to be regenerated.)